packages/migrate/migrate.ts

```diff
@@ -0,0 +1,112 @@
+import jscodeshift from "jscodeshift";
+import pEachSeries from "p-each-series";
+import PLazy from "p-lazy";
+
+import loadersTransform from "./loaders/loaders";
+import resolveTransform from "./resolve/resolve";
+import removeJsonLoaderTransform from "./removeJsonLoader/removeJsonLoader";
+import uglifyJsPluginTransform from "./uglifyJsPlugin/uglifyJsPlugin";
+import loaderOptionsPluginTransform from "./loaderOptionsPlugin/loaderOptionsPlugin";
+import bannerPluginTransform from "./bannerPlugin/bannerPlugin";
+import extractTextPluginTransform from "./extractTextPlugin/extractTextPlugin";
+import noEmitOnErrorsPluginTransform from "./noEmitOnErrorsPlugin/noEmitOnErrorsPlugin";
+import removeDeprecatedPluginsTransform from "./removeDeprecatedPlugins/removeDeprecatedPlugins";
+
+interface ItransformsObject {
+	loadersTransform: object;
+	resolveTransform: object;
```

Comment:
Actually these are modules, maybe either declare them as modules ( if possible )

Reply:
You mean [TS modules](https://www.typescriptlang.org/docs/handbook/modules.html)? Also, can you please show an example of what would be inside each module?